src/extension.ts

```diff
@@ -12,6 +12,21 @@ import { IValue } from "./Model";
 import { Utils } from "./Utils";
 import { VSCodeUI } from "./VSCodeUI";
 
+class Step {
+    public readonly name: string;
+    public readonly info: string;
+    constructor(name: string, info: string) {
+        this.name = name;
+        this.info = info;
+    }
+}
+const stepGroupId: Step = new Step("GroupId", "GroupId inputed.");
+const stepArtifactId: Step = new Step("ArtifactId", "ArtifactId inputed.");
+const stepBootVersion: Step = new Step("BootVersion", "BootVersion selected.");
+const stepDependencies: Step = new Step("Dependencies", "Dependencies selected.");
+const stepTargetFolder: Step = new Step("TargetFolder", "Target folder selected.");
+const stepDownloadUnzip: Step = new Step("DownloadUnzip", "Package unzipped.");
```

Comment:
Are these variables only be used in `activate()`? If yes, I prefer putting them inside the method scope.

while putting them as global variable is alright though, leaving the choice to yourself.